group_vars/all.yml

```diff
@@ -134,8 +134,8 @@ nginx_sites_available:
 
   fairfood_https:
     - |
-      listen 443;
-      listen [::]:443;
+      listen 443 ssl;
+      listen [::]:443 ssl;
```

Comment:
Why do we disable the default site?

Reply:
I recently added the default site to the new server, in order to catch disable any requests for the non-primary domain. That is, we want to respond only to members.ceresfairfood.org.au, not prod3.ceresfairfood.org.au (mainly as a surefire way to avoid search engines crawling the wrong domain).

But during the switchover, the server needs to respond to prod3.ceresfairfood.org.au (or perhaps IP address) for proxy passing. So I disable the default server temporarily to allow this.

Reply:
Why don't you add prod3 to the app config for the switchover? Alternatively, what do you think about IP plus header?
```
    proxy_pass https://43.239.97.148;
    proxy_ssl_name $host;
    proxy_set_header Host $host;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

The fewer changes we need on the new production server, the better.

Reply:
Ah, yes of course that is much neater!